packages/cardano-services/src/ChainHistory/DbSyncChainHistory/queries.ts

```diff
@@ -258,6 +258,8 @@ export const findPoolRegisterCertsByTxIds = `
 	FROM tx
 	JOIN pool_update AS cert ON cert.registered_tx_id = tx.id
 	JOIN pool_hash AS pool ON pool.id = cert.hash_id
+	JOIN block ON block_id = block.id
+	JOIN epoch_param ON epoch_param.epoch_no = (CASE WHEN block.epoch_no > 1 THEN block.epoch_no ELSE 1 END)
```

Comment:
I removed this to optimize the queries with newly added explicit certificate deposits in newer db-sync versions.

Looking at the selected fields list it seems this is now redundant. If CI tests passes, probably this should be removed.

Reply:
Done. I squashed the commit into the merge